content/07.pathogenesis.md

```diff
@@ -46,8 +46,12 @@ One severe example includes reports of cytokine storm-like responses in patients
 #### Reproduction Number and Dynamics of Transmission
 
 Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
-The basic reproduction number, R~0~, is the expected number of new infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
+In models the quantities of basic reproduction (or reproductive) number and the timescale of infection have a large influence on the population level dynamics of epidemics.
+The basic reproduction number, R~0~, is the expected number of new (secondary) infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
+R~0~ is a unitless number which in models is a combination of parameters principally related to the transmission rate (rate of infection transmitting interactions for an infectious individual) and the infectious period.
```

Comment:
```suggestion
R~0~ is a unitless number which is calculated from data on transmission rates (based on the number of interactions between infectious and susceptible individuals) and the infectious period.
```

Reply:
Should we also mention that the likelihood of an interaction resulting in a new infection is another major determining factor?

Reply:
Is R0 actually _calculated_ this way? I just assumed that infections/interaction and interactions/unit-time were too difficult to estimate to make calculating R~0~ from these feasible... If it's not calculated this way maybe replace "calculated" with "equivalent to" or something like that?

Reply:
This is why the original sentence as I added had the detail "in models".  At least for some pathogens/epidemics, it's easier to measure the infectious period & the R0 & then derive the transmission rate by the relation of R0 ~ transmission rate / infectious period. (I'll go through the whole thing again to try to make this pull request's components better b/c so much quick changing of math/semantic details is making it messy. I tried to add sentences that went along with the current structure without modifying other people's sentences scientifically/mathematically unless there was something really obviously unclear which there wasn't for this section or in general. I could have done it better though by inserting more of the references or holding off on more complicated topics.)

Reply:
After looking at this again, I saw where the ambiguity is coming from, @alavendelm @rdvelazquez. I meant "in mechanistic models" (distinguishing between models which are more derived from biological mechanisms and models which incorporate less biological mechanisms while comparing to data and sort of in between models like those that are sometimes used to derive R from early epidemic data). (Also note, in a mechanistic model, transmission rate could be broken down into smaller details such as what @rdvelazquez  mentioned, so the sentence already covers the smaller details)  Here's a suggested better sentence incorporating a few of the references as well:
"R~0~ is a unitless number which in mechanistic models is a combination of parameters principally related to the transmission rate (e.g., rate of infection transmitting interactions for an infectious individual) and the infectious period [@DOI:10.1126/science.abb5659][@ISBN:9780691116174]. "
If this resolves all the comments then I'd appreciate if someone can put in this better sentence.